eureka-resources/src/main/resources/css/main.css

```diff
@@ -146,7 +146,7 @@ tr.odd {
 span.hlist {
   padding-right: 10px;
   margin-right: 10px;
-  #background-color: #ffffcc;
+  background-color: #ffffcc;
```

Comment:
Let's keep the existing background-color please. Can just delete this line.

Reply:
Done